etl/etl/competition.py

```diff
@@ -1595,3 +1557,61 @@ def cell(self, proposal, column_name):
                 return split_name[0]
             elif len(split_name) > 1:
                 return split_name[1]
+
+class ColumnSplitter(InformationTransformer):
```

Comment:
Do we still need this class?

Reply:
I'll take it out -- it isn't used any longer, though it's a more generic splitter than the "name splitter" so it might be good to keep it and plan to transition to it?

Happy to take it out (we can always add it back if we ever wanted to do that mini-refactor later)